src/main/resources/plugin.yml

```diff
@@ -1,21 +1,82 @@
 name: AnarchyExploitFixes
 version: ${project.version}
-main: me.moomoo.anarchyexploitfixes.Main
+main: me.moomoo.anarchyexploitfixes.AnarchyExploitFixes
+prefix: AnarchyExploitFixes
```

Comment:
Prefix should be plugin name by default.

Reply:
Also thanks